app/views/steps/check/results/show.en.html+multiples.erb

```diff
@@ -53,15 +53,15 @@
     <p class="govuk-body">
       When a caution or conviction is spent you do not need to tell people about it, or the offence that led to it,
       if they’re carrying out a
-      <a class="govuk-link" rel="external" target="_blank" href="https://www.gov.uk/guidance/telling-people-about-your-criminal-record#basic-dbs-checks">basic
+      <a class="govuk-link" rel="external" target="_blank" href="www.gov.uk/tell-employer-or-college-about-criminal-record/what-information-you-need-to-give">basic
```

Comment:
Hey super minor thing but I think is better to leave the `https://`
It helps a lot when having to search globally for links 😄

Reply:
Ups!

Reply:
I was given the links in an image, was too lazy to type the whole url with https... then copied it all here ... fixing this!